<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWidget</class>
 <widget class="QWidget" name="ClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditport">
     <property name="text">
      <string>8888</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="pushButtonclose">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>217</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>服务器IP</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2" colspan="2">
    <widget class="QLineEdit" name="lineEditip">
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="pushButtonsend">
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>服务器端口</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QTextEdit" name="textEditread"/>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QTextEdit" name="textEditwrite"/>
   </item>
   <item row="0" column="3" rowspan="3">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
